modal/image.py

```diff
@@ -465,6 +467,15 @@ async def join():
         obj.force_build = force_build
         return obj
 
+    async def build(self, app: "_App", client: Optional[_Client] = None) -> None:
+        """Build the image immediately rather than waiting for it to be used."""
+
+        client = client or app._client or await _Client.from_env()
```

Comment:
Do we need `app` as an argument here? Feels confusing to have an object that is usually passed to `App` have a method that accepts an `App` as an argument.